modules/olm-creating-catalog-from-index.adoc

```diff
@@ -6,10 +6,12 @@
 ifdef::openshift-origin[]
 :index-image: catalog
 :tag: latest
+:namespace: olm
```

Comment:
This is great!

Reply:
Didn't mean to submit this, was a test review I was showing in a new hire working group meeting. :sweat_smile: But I can't delete it... ??